tests/unit/views/notifications/organization_admin_notification.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Organization Admin Notification
+  As an Organization admin, I want to be notified when another admin
+  is added, removed or has their role changed.
+
+  Background:
+    Given Vetinari is an owner of the Ankh-Morpork organization
+    And Vimes is an admin of the Ankh-Morpork organization
+
+  Scenario Outline: Vetinari adds Ridcully
+    When Vetinari adds Ridcully as <owner_or_admin>
+    Then <user> gets notified with <notification_string> about the addition
+
+    Examples:
+      | user     | owner_or_admin | notification_string                                           |
+      | Vetinari | owner          | You made Mustrum Ridcully owner of Ankh-Morpork               |
+      | Ridcully | owner          | Havelock Vetinari made you owner of Ankh-Morpork              |
+      | Vimes    | owner          | Havelock Vetinari made Mustrum Ridcully owner of Ankh-Morpork |
+      | Vetinari | admin          | You made Mustrum Ridcully admin of Ankh-Morpork               |
+      | Ridcully | admin          | Havelock Vetinari made you admin of Ankh-Morpork              |
+      | Vimes    | admin          | Havelock Vetinari made Mustrum Ridcully admin of Ankh-Morpork |
+
+  Scenario Outline: Vetinari invites Ridcully
+    When Vetinari invites Ridcully as <owner_or_admin> to the Ankh-Morpork organization
```

Comment:
"… _of_ the…", but maybe we don't need to repeat "the Ankh-Morpork organization" each time? As long as the `@given` declarations don't cause problems in other tests, leaving out redundant text will improve readability.